runelite-client/src/main/java/net/runelite/client/plugins/zoom/ZoomConfig.java

```diff
@@ -54,10 +56,22 @@ default boolean innerLimit()
 	@ConfigItem(
 		keyName = "relaxCameraPitch",
 		name = "Vertical camera",
-		description = "Relax the camera's upper pitch limit"
+		description = "Relax the camera's upper pitch limit",
+		position = 3
 	)
 	default boolean relaxCameraPitch()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "controlZoom",
+		name = "Control Zoom",
```

Comment:
I would name this like "Require Control down" or something, "Control Zoom" seems weird